myeongseok/app.js

```diff
@@ -98,6 +98,17 @@ app.get('/posts', (req, res, next) => {
   );
 });
 
+app.delete('/posts/:postId/users/:userId', async (req, res) => {
```

Comment:
```suggestion
app.delete('/posts/:postId', async (req, res) => {
```
user에 대한 정보는 주소에 담기지 않아도 상관없습니다. 어차피 post 하나를 지울 때 필요한 데이터는 postId 하나 뿐입니다.

만약 권한을 생각해서 이렇게 작성하신거라면 그 부분은 `인증 & 인가` 세션에서 자세히 다룰 예정이니 우선은 가볍게 넘어가주시면 되겠습니다~

Reply:
네 postId로만 요청을 받으면, 임의의 제3자의 요청이 우려되어서 추가했었습니다. 해당 내용수정하겠습니다!